### ISSUE TITLE
Add branches to CI

### ISSUE THREAD

**Issue by colinsheppard** [LBNL-UCB-STI/beam#32]
Please enable these branches to build with Travis CI in addition to master. Let me know if there's anything I need to configure on github.

agentsim
router
physsim
integration

**Comment by zishanbilal** [comment]
Its just to add .travis.yml in the specific branch, by default all branches lie under CI. I merged master in the above mentioned branches, and now the build for all of these branches is passing.

To check the status of all branches we can use [this](https://travis-ci.org/LBNL-UCB-STI/beam/branches) link from Travis site. I have also created a markdown for build status of feature branches as well.

But merging master or any of the feature branch into task branches may bring undesired changes that would cause some problem for the task owner. We may cherry-pick required commits to the desired branch. By this all active branches of beam would be under Travis CI. I cherry-picked using `git cherry-pick 7dd9a93^..271461d` into router-r5-extractor and one can use the same for his branch as well.
